tools/unsmile.c

```diff
@@ -27,7 +27,7 @@
 #include "usage.h"
 #include <smile.h>
 
-#define BUFFER_SIZE 65536
+#define BUFFER_SIZE 32768
```

Comment:
I'm confident we don't need buffers quite so large, given the size of what we're parsing.